scripts/compaction.sh

```diff
@@ -0,0 +1,17 @@
+lineOfKeyspaces=`./service/bin/cqlsh -e "describe keyspaces; "`
+echo $lineOfKeyspaces
+
+for keyspace in ${lineOfKeyspaces[@]}
+do
+        echo "keyspace = '$keyspace'"
+        keyspaceWithoutQuotes="${keyspace//\"}"
+        echo "keyspaceWithoutQuotes = $keyspaceWithoutQuotes"
+        if [[ $keyspace == *"system"* || $keyspace == *"__simple_rf_test_keyspace__"* ]]; then
+                echo "Skipping $keyspace as this is a test or system keyspace!"
+                continue;
+        fi
+        txTable=" _transactions"
+        ./service/bin/cqlsh -e "use $keyspace; alter table \"_transactions\" with gc_grace_seconds = 0;"
```

Comment:
Should we reset the gc_grace_seconds once we're done?

Reply:
Doesnt really matter, we can also skip the alter gc_grace_seconds as the required time has passed, but it hadnt on my test stack and this is the script tested.

Reply:
We think manual changes to the `gc_grace_seconds` is a possible reason for Atlas failing to start against a degraded cluster.  So I'd push for not changing it if we can avoid it.

Reply:
Lets skip it then.

Reply:
Yea I'd rather avoid altering `gc_grace_seconds` as well.